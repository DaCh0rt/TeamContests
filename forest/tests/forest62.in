627 524
-4879 7382
-3720 8543
5588 -5288
-4367 -5145
-2029 -9066
-201 -2820
3376 -5972
6479 5583
566 7203
-9616 -6349
-5674 -4362
1966 -7824
6903 -3230
-1172 -3077
6657 -9299
3195 1728
2475 -8100
-5784 3883
4309 5412
154 4951
6896 -4723
-50 -1642
-8650 7508
-6025 -469
-9045 1962
5173 7724
-3293 2791
-4339 -9631
-8278 -9439
1000 1939
-9463 -2473
9771 3070
7502 3733
-8479 -4741
-8391 -9476
7215 8812
6639 8956
2521 -970
-5698 -7661
-9001 -2996
4695 -6546
1147 3661
-6188 -6811
881 -7450
9627 7362
-6369 -1148
7497 -4241
-720 9205
-6320 3752
3372 8019
2860 -1938
-6800 7227
730 1629
980 -3173
-3583 4788
8113 -5637
-6309 -3473
8525 1753
-3008 -9025
-6155 -9635
-7851 1140
-4212 -4736
5065 8458
5845 9680
-571 8722
-726 -27
-525 -5862
9222 -2538
-5924 -6738
7273 9683
805 1519
-1224 -6108
3260 3298
3428 1253
-2960 -2437
-1293 -6997
8820 -1341
4028 4237
2431 -8149
-9337 9216
9853 4513
2096 -7796
-8514 -9075
6511 9852
4249 -5705
-7649 -2023
9840 -9286
9681 5136
1331 8213
-4856 -2026
2188 5029
-934 2377
-420 3834
3816 -4935
-396 5151
1903 6640
-8003 -7612
-7325 -9614
-4763 -6797
-7978 7625
9835 8383
4406 -392
-3685 -1715
-7874 -8090
1457 -7391
-1565 -9752
1823 105
5018 6103
4811 -6414
3161 39
2575 6626
2985 -7917
2717 -2507
-6659 1553
-2792 4682
2724 -6166
2104 6278
-5942 9525
6592 5440
6499 2544
-4176 767
9961 -4717
-1269 2758
6513 -3808
-2773 -1603
2404 1439
9929 -4776
-4677 904
-8698 -4554
-7003 -1311
-5748 1272
-122 -8863
-7268 -6644
-1144 4623
3317 -2169
1030 2199
7883 9308
1820 -9188
-9873 1966
4594 8659
-6813 9519
-4205 6009
7603 -1768
3228 -2131
5747 4676
-5973 -2301
-569 -6830
8737 -2133
-4507 -3557
-5179 1562
9148 6214
-5537 -2062
-4848 -6343
7738 2038
6493 -5757
-4509 -6313
5010 3386
-4528 9555
-6323 -8776
2849 2539
5254 -3306
6479 -2373
7691 -9609
-3837 7353
-3526 6707
6974 -6039
9288 3537
1082 5572
-4769 1987
8906 -1443
5483 889
8998 -6323
-1646 4250
5000 571
4582 -1282
-8654 -4909
-6713 -9408
-1538 8074
-4289 2225
1304 1596
7868 6429
-5023 7380
-6455 5982
-7150 5507
2430 1876
-7312 1570
-8648 -6874
-9633 -5991
9342 2723
3581 5273
8506 -3547
-6056 2732
-3340 -2718
6857 -888
2286 -4439
-3343 -8901
4892 5883
5079 3898
1170 5006
7386 -9829
7386 -4703
3972 1671
-5003 -9111
8937 -5922
-2932 -3360
-2841 -9086
3384 9871
-2001 7809
-2591 9668
-6383 -7390
3708 580
-5204 7524
1746 8982
-9285 4405
890 -8130
3377 -4829
8000 -4847
4519 8304
8390 -2016
6205 3274
-4936 7985
1268 183
7988 183
-8047 -8928
-3117 -4733
-9212 -2502
-1142 3685
5010 -9282
-9061 -4878
-6976 -9619
-2260 -4402
4669 -2847
9454 3236
207 -5913
-122 -9581
-5906 -6125
6277 6737
-9467 -904
-373 -9484
-6949 -8267
-943 -8512
-6621 -4935
428 2837
8159 -9409
7421 4373
-6046 2997
-3803 9013
-9318 -3019
1873 -7775
7539 2605
7124 5624
1241 8170
1155 -8463
4106 -7624
-9727 -7028
-4193 -1903
6977 -3842
2787 -6984
4056 -3512
9376 461
1407 2028
523 -9942
1762 4456
9452 3522
-8127 732
-1218 -1504
5315 -1552
1996 -1638
-9892 9850
4132 506
-6048 303
-8759 -5593
-265 6723
3360 -4855
5388 -7820
6190 7020
-8619 -7850
7408 -5268
-3492 -1923
260 -8865
-5270 -6485
-1799 3708
3197 -5580
-5151 -647
-2136 -9985
7321 3133
-7473 5712
-9016 -4609
-9735 -8810
3356 8121
1169 9617
5150 -4152
-3465 -5708
-3463 -8067
6981 1723
2545 8554
6404 3208
-8645 2525
-2252 8801
-8084 3578
-8368 5448
3349 -446
9254 -1445
-5214 -6370
1511 4612
2667 121
859 -6810
733 -5927
-9655 -8370
-631 4473
-7192 -9411
-5838 -1812
-1516 -4653
-3836 9656
-9272 -112
9894 -9575
653 -4270
5585 -8900
-9297 -1591
7791 -5229
2622 -7839
4073 -6854
-2702 2364
667 7594
-9051 8567
-4172 -7396
8083 -5399
5845 -1997
-6305 4514
-1776 9216
3113 -5233
-1168 931
189 -2701
6527 8744
-3998 -4213
3030 7399
747 -9181
9468 3972
9261 3489
-1003 -7344
2414 4620
-1986 -5610
4540 9714
-6234 -6261
-4716 4039
-1166 5691
-4294 -4385
-6452 -2438
-4807 4628
8256 6608
-8624 -7434
5172 -1333
333 -1935
7236 2299
9333 5333
7962 -3098
2285 -4142
-9121 147
-3737 6138
2949 -1824
1547 4348
8704 5171
-4319 5100
-5864 3797
-562 1074
1712 5244
-9204 -8598
-4226 1106
7506 452
-8864 888
-7776 -2559
6077 4178
-407 -310
-6367 -9137
-3032 -3645
-5909 18
9518 9319
402 4114
-2474 -7192
-3263 -1389
1951 -4370
-1133 -7217
-2163 5536
-3168 -6706
7118 3729
6838 -6297
2984 -5894
-1595 -2503
7529 -9895
-3592 -7118
-2674 7177
4755 8858
1641 2668
-4750 -6939
6413 -3973
7829 149
7147 5829
-6827 2104
6703 7325
4586 -4830
1902 8869
-844 -5500
-7522 4221
1993 -2746
3444 -9953
5372 -3459
7778 4373
4510 -5857
-399 -8720
-9432 4075
9132 9218
-8438 -1920
-2119 6572
4518 3934
-5780 9982
-6436 9470
-9805 5289
9423 9476
1857 -2963
5903 9224
-2254 -1970
358 -8645
-6126 2398
7526 -7683
-7043 4589
2337 -4219
-9057 3248
-7919 -5848
9647 -2193
-9101 1198
-2773 -9237
-8793 9870
-5916 2826
-1094 2696
3779 2046
-3828 -8824
2518 2428
-2139 -9768
3033 -3891
2454 1555
-903 -4560
8925 -2842
4404 -5769
9424 -3576
-7751 7432
3768 6335
-6634 5153
3278 -8208
-6437 3049
-7477 7154
9637 8625
-7106 9179
7250 -5138
9352 5016
3701 2859
8390 -5475
3430 8751
5971 -5419
-5238 4934
-3511 -8280
-9800 -1100
-8418 7112
9664 4786
-8944 -6740
7288 5603
1426 8854
-9271 9718
-5811 5846
1313 -9025
8624 8432
3243 -2055
7920 -9544
5311 -3621
-7653 -6394
4321 1696
-8223 9706
3524 -9211
-2896 -5559
8565 -3721
232 -5668
7722 7015
9220 -1529
-3602 873
-5678 8846
7691 -897
6860 3790
-2466 -7176
7350 5496
-8725 -7473
-6464 -2780
-5197 7318
-643 -5187
2622 -9149
4232 1519
-2257 3227
-1313 2680
8690 -7175
-8288 -7180
-7766 2832
1880 8684
-2577 3738
-3701 5325
-8352 -6699
648 9791
1247 -4667
-279 -1508
-9727 2799
-9795 8521
4484 -2683
9068 9814
-5732 -7263
4808 9456
-5030 1417
-4792 6716
6052 9558
-6581 5436
-2930 -6235
-3713 -3141
-8511 -6141
6581 3415
3461 5439
-6134 -1778
-4837 -8786
-3543 6263
7980 -4128
-613 7359
4066 5690
1795 -5085
835 -8485
7542 -9953
-4596 9925
-9322 2207
-1473 -2235
-6643 -1996
-6124 5856
-6743 -2219
-2720 -2312
-8869 -9761
-5800 7787
7803 -9864
5998 -2467
-188 -9870
-5476 1785
-9079 -2734
-8989 9362
6163 9442
-7389 5930
292 -9405
-728 3775
1385 8096
-3613 8367
-4844 933
9210 8129
7063 6326
6512 3817
6941 -6595
3503 -9346
-3898 2809
-2102 -9926
9066 -9489
690 -7206
-164 8878
-5008 846
3882 -6703
-3031 3451
-9776 -5148
8802 -4765
4913 7568
-3226 7924
-7645 -9025
6447 -2753
-7392 -6372
3431 -8135
-9595 -7277
2833 6689
1925 -4469
474 -9073
-2169 -7764
1906 9462
-7777 -9087
587 -3399
-7142 -2707
-3382 -649
9385 8077
6385 -9119
-3314 -365
2624 -5257
-2586 -6468
3099 3519
6799 6959
-3235 -1438
7055 504
-4390 -7839
-3041 4175
2269 -1608
1327 9183
4126 -4050
709 6634
1724 -1119
-813 6690
-4097 -3442
4547 9335
-104 -738
-3090 5394
8011 -2403
-3116 -7871
7989 5296
-8804 -8596
3454 9246
-8461 -35
-6049 4606
4627 7644
-4835 -6148
-7435 1151
-5134 -3434
9782 -194
5023 -4109
961 -2295
-4154 9316
-4197 7542
6042 -4324
-1801 442
3691 -1924
2564 -7803
1509 305
4503 -6119
172 -3384
