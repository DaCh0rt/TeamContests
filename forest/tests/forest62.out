80620.53 431343261.89
